src/RadioGroup/RadioGroup.less

```diff
@@ -5,7 +5,9 @@
     .margin--top--l;
   }
 
-  .RadioGroup--radio {
+  // The latter class (selectedWithAdditionalText) exists purely to be overriden by the consumer.
+  .RadioGroup--radio,
+  .RadioGroup--radio--selectedWithAdditionalText {
```

Comment:
Ideally `.RadioGroup--radio--selectedWithAdditionalText` would simply be its own empty class - so that a consumer could override it if desired, but it would otherwise not affect the component's appearance or functionality whatsoever - but we don't generally allow those.

This seems like a reasonable compromise which I wouldn't expect to break styling anywhere? Open to other ideas.

Reply:
This seems okay to me! I'm not sure what ends up needing to be overridden once this gets imported into fampo but I think when I tried the first time I had to override the `.margin--top--xs;` in line 24 below across the board in fampo (although I hadn't needed to when I was testing by copying the components dist folder over). But I think I added that line in the first place to make it look right in Dewey. 

Want to check if you're seeing the same thing because it's possible that that top margin isn't actually needed? (if it just gets overridden every time elsewhere)
Basically it felt a little fishy after I actually imported it info fampo so just want to check if you think it's still necessary

Reply:
That margin top _does_ actually seem to make a difference in Dewey, so I'll leave it alone! Don't wanna alter the component just to suit our use case in a way that's not part of its inherent design in Dewey after all, even if it's been a bit hard to tell which things are 'issues' and which are just custom needs for our current use case in fampo. 😅